man/portage.5

```diff
@@ -1455,6 +1455,14 @@ listed toward the left of the list.
 .BR repo\-name " = <value of profiles/repo_name>"
 The name of this repository (overrides profiles/repo_name if it exists).
 .TP
+.BR properties\-allowed
+List of PROPERTIES tokens which are allowed for ebuilds within
+the repo.  If unset, all tokens are allowed.
```

Comment:
```suggestion
the repo.  If unset, any token is allowed.
```

I think this way we make it more clear that we allow arbitrary token, not just some well-defined set.